### PR TITLE
Fix thrust equations

### LINKED ISSUE #383: Doc Bug: Land Detector: thrust equations
The equations for the thrust conditions of "ground contact" and "maybe landed" don't seem to correspond to the ones in the code.

They should be:

`MPC_THR_MIN + (MPC_THR_HOVER - MPC_THR_MIN) * 0.3f`

and

`MPC_THR_MIN + (MPC_THR_HOVER - MPC_THR_MIN) * LNDMC_THR_RANGE`

respectively.

Bug Page: [Land Detector · PX4 Developer Guide](https://dev.px4.io/en/tutorials/land_detector.html)

### PR DESCRIPTION
**Pull request by hamishwillee**
Fixes #383. Fix a couple of typos. 

@ndepal Anything else to add here?

### CODE REVIEW

File: en/tutorials/land_detector.md [hunk 1]
```diff
@@ -10,46 +10,48 @@ By default the land detector does detect landing, but does not auto-disarm. If t
 
 The complete set of parameters is available in the *QGroundControl* parameter editor under the `LNDMC` prefix. The key parameters that might differ per airframe are these:
 
-* [MPC_THR_HOVER](../advanced/parameter_reference.md#MPC_THR_HOVER) - the hover throttle of the system \(in percent, default is 50%\). It is important to set this correctly as it does not only make the altitude control more accurate, but also ensures correct land detection. A racer or a big camera drone without payload mounted might need a much lower setting \(e.g. 35%\).
+* [MPC_THR_HOVER](../advanced/parameter_reference.md#MPC_THR_HOVER) - the hover throttle of the system \(in percent, default is 50%\). It is important to set this correctly as not only it does it make the altitude control more accurate, but it also ensures correct land detection. A racer or a big camera drone without payload mounted might need a much lower setting \(e.g. 35%\).
```
Comment (ndepal) [inline en/tutorials/land_detector.md]: >It is important to set this correctly as not only ~it~ does it make

There is one too many "it"s now.

File: en/tutorials/land_detector.md [hunk 2]
```diff
@@ -10,46 +10,48 @@ By default the land detector does detect landing, but does not auto-disarm. If t
...
 This state is reached if following conditions are true for 0.35 seconds:
 
 - no vertical movement ([LNDMC_Z_VEL_MAX](../advanced/parameter_reference.md#LNDMC_Z_VEL_MAX))
 - no horizontal movement ([LNDMC_XY_VEL_MAX](../advanced/parameter_reference.md#LNDMC_XY_VEL_MAX))
-- low thrust `(0.3 * MPC_THR_MIN + LNDMC_THR_RANGE* (MPC_THR_HOVER-MPC_THR_MIN))` or velocity setpoint is 0.9 of land speed but vehicle has no vertical movement.
+- low thrust `MPC_THR_MIN + (MPC_THR_HOVER - MPC_THR_MIN) * 0.3f` or velocity setpoint is 0.9 of land speed but vehicle has no vertical movement.
 
-If the vehicle is in position or velocity control and ground contact was detected, 
-the position controller will set the thrust vector along the body x-y-axis to zero. 
+If the vehicle is in position or velocity control and ground contact were detected,
```
Comment (ndepal) [inline en/tutorials/land_detector.md]: "was" was correct here.

File: en/tutorials/land_detector.md [hunk 3]
```diff
@@ -10,46 +10,48 @@ By default the land detector does detect landing, but does not auto-disarm. If t
...
 
-1. Ground contact
+#### Ground contact
 
 This state is reached if following conditions are true for 0.35 seconds:
 
 - no vertical movement ([LNDMC_Z_VEL_MAX](../advanced/parameter_reference.md#LNDMC_Z_VEL_MAX))
 - no horizontal movement ([LNDMC_XY_VEL_MAX](../advanced/parameter_reference.md#LNDMC_XY_VEL_MAX))
-- low thrust `(0.3 * MPC_THR_MIN + LNDMC_THR_RANGE* (MPC_THR_HOVER-MPC_THR_MIN))` or velocity setpoint is 0.9 of land speed but vehicle has no vertical movement.
+- low thrust `MPC_THR_MIN + (MPC_THR_HOVER - MPC_THR_MIN) * 0.3f` or velocity setpoint is 0.9 of land speed but vehicle has no vertical movement.
```
Comment (ndepal) [inline en/tutorials/land_detector.md:34]: :+1:

File: en/tutorials/land_detector.md [hunk 4]
```diff
@@ -10,46 +10,48 @@ By default the land detector does detect landing, but does not auto-disarm. If t
...
-2. Maybe landed
+
+#### Maybe landed
 
 This state is reached if following conditions are true for 0.25 seconds:
 
 - all conditions of ground contact are true
 - is not rotating ([LNDMC_ROT_MAX](../advanced/parameter_reference.md#LNDMC_ROT_MAX))
-- has low thrust `(LNDMC_THR_RANGE * MPC_THR_MIN + LNDMC_THR_RANGE* (MPC_THR_HOVER-MPC_THR_MIN))`; 
+- has low thrust `MPC_THR_MIN + (MPC_THR_HOVER - MPC_THR_MIN) * LNDMC_THR_RANGE`;
```
Comment (ndepal) [inline en/tutorials/land_detector.md:46]: :+1:

**Review by ndepal — CHANGES_REQUESTED** [review]
Looks good. The equations are fixed. Noticed a couple of language mistakes.

File: en/tutorials/land_detector.md (hunk 2 above)
Comment (hamishwillee) [reply en/tutorials/land_detector.md]: Ah, I read this as: 
- If the vehicle is in position
- or velocity control and ground context were detected
I'll make it less ambiguous by making:

"If the vehicle is in position- or velocity- control and ground contact was detected, "

File: en/tutorials/land_detector.md (hunk 1 above)
Comment (hamishwillee) [reply en/tutorials/land_detector.md]: Thanks!

**Comment by hamishwillee** [comment]
Thanks for review and original issue @ndepal - much appreciated. I fixed your comments and this is now merged.

**Pull request merged**